benchmarks/6.1.0/v2/6.1.0_single-node_4-core_jdk-11.md

```diff
@@ -0,0 +1,168 @@
+# V2
+
+The the newly introduced features/improvements to Identity Server 6.1 base product were being used when getting these performance results. Please refer to the [release note](https://is.docs.wso2.com/en/latest/references/about-this-release/) for further information.
```

Comment:
In this version of Performance results, the tests were executed against the default 6.1.0 base product without enabling or disabling any additional features.